packages/cacti-copm-core/src/main/protos-kt/out/org/hyperledger/cacti/plugin/cacti/plugin/copm/core/services/defaultservice/DefaultServiceOuterClassGrpcKt.kt

```diff
@@ -0,0 +1,314 @@
+package org.hyperledger.cacti.plugin.cacti.plugin.copm.core.services.defaultservice
```

Comment:
@jenniferlianne If this is generated code then please make sure to have that represented somewhere along the directory path with a path element of `.../generated/...` so that it's always clearly distinguishable from hand-maintained code. It helps during reviews and also whenever we add more automation it's extremely helpful while constructing globs/regexes that find or exclude generated code from automation such as linting.

Reply:
This was committed by mistake.  Thanks.